app/forms/field_handlers/select_handlers.py

```diff
@@ -18,7 +20,7 @@ def coerce_str_unless_none(value):
         return str(value) if value is not None else None
 
     @staticmethod
-    def build_choices_with_detail_answer_ids(options: dict):
+    def build_choices_with_detail_answer_ids(options: dict) -> Sequence[Any]:
```

Comment:
This returns a list of tuples of strings. We should use concrete types when returing i.e list over Sequence, but Sequence over list when it is an argumen. https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/doc/python-type-hinting.md#abstract-vs-concrete

Reply:
Using list now. Thanks!